app/views/admin/enterprises/form/_primary_details.html.haml

```diff
@@ -63,6 +63,13 @@
       %span{ ng: { class: 'availability.toLowerCase()', hide: "checking" } }
         {{ availability }}
         %i{ ng: { class: "{'icon-ok-sign': availability == 'Available', 'icon-remove-sign': availability == 'Unavailable'}" } }
+  .row{ ng: { show: "Enterprise.sells == 'own' || Enterprise.sells == 'any'" } }
```

Comment:
I'd say if enterprise sells none you also want to have the id anyway.

Reply:
Ah yes, I just copied the pattern from `permalink` row. I will remove these options.